robustus/detail/install_opencv.py

```diff
@@ -59,13 +59,11 @@ def install(robustus, requirement_specifier, rob_file, ignore_index):
         def in_cache():
             return os.path.isfile(cv2so)
 
-        versions_to_fix_rpath = ['2.4.7', '2.4.8', '2.4.9']
```

Comment:
@olegsinyavskiy do you remember why we fix rpath only for specific opencv versions? it seems like these are all the version we use.